test/other/embind_tsgen_ignore_2.d.ts

```diff
@@ -6,6 +6,7 @@ type EmbindString = ArrayBuffer|Uint8Array|Uint8ClampedArray|Int8Array|string;
 export interface Test {
   x: number;
   readonly y: number;
+  get stringProperty(): string;set stringProperty(value: EmbindString);
```

Comment:
Can you remind me what EmbindString is?

Reply:
It's alias for all the types that embind can convert to std::strings
https://github.com/emscripten-core/emscripten/blob/7f509d5320528ff31731450d87b9f560dbaf799d/test/other/embind_tsgen.d.ts#L19